README.md

```diff
@@ -1 +1,17 @@
-# github-status
\ No newline at end of file
+# github-status
+
+## Deployment
+Build Docker container with:
+
+`docker build -f ./Dockerfile -t githubstatus .`
+
+Then run it like this:
+
+`docker run --env BOT_TOKEN=<token> --env CHAT_ID=<chat_id> githubstatus`
+
+## Run flake8
```

Comment:
Let's include testing part to the next commit. So this commit should disappear.

Reply:
Fixed